### PR TITLE
First version of documentation page

### PR DESCRIPTION
**Pull request by AlecAivazis**

### CODE REVIEW

File: docs/src/views/Basics/GettingStarted/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <Description textStyle={{ marginBottom: 0 }}>
+            Getting started with quark is as easy as installing the version that is appropriate for
+            your application, either run `npm i quark-web` or `npm i quark-native`. Once you have
+            the package locally, there are a few components that you should be aware of. First is
+            `App` which should act as the root component of your application. It will make sure that
+            your app is always at least as big as your viewport, allowing you to easily create
+            layouts, using the `FlexRow` and `FlexColumn` components.
+        </Description>
+        <Description textStyle={{ marginBottom: 0 }}>
+            Once you have your application wrapped in an App, and you have gotten used to using the
```
Comment (meinstein) [inline docs/src/views/Basics/GettingStarted/index.js]: missing `App` highlighting

**Review by meinstein — CHANGES_REQUESTED** [review]
The font-family is not getting applied to the `Basics` and `Design` sections for some reason...
screen shot 2018-04-27 at 7 50 30 pm

File: docs/src/views/Basics/Introduction/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            Quark is a component library designed to be used in both `React` and `ReactNative` in
+            order to provide one single vocabulary for building interfaces. Along with common visual
+            components, it also has powerful state management components, useful for quickly
+            building complex state in a composable manner.
+        </Description>
+        <Description textStyle={{ marginBottom: 0 }}>
+            To get started, please read the `Getting Started` guide on the left.
+        </Description>
+    </FlexColumn>
+)
```
Comment (meinstein) [inline docs/src/views/Basics/Introduction/index.js]: I think it's worth calling out here that components that cannot be universally applied in both Web and Native are tagged accordingly.

**Comment by AlecAivazis** [comment]
*sigh* took a depressingly long time to figure out why the netlify build wasn't working but I got there eventually. I fixed the styling issue with the headers @meinstein, thanks for finding them. I'm going to address the other issues you brought up 👍

**Comment by meinstein** [comment]
What do you think about adding some borders to the rows? The bigger tables, especially with lots of non-nullable and non-required props, are tough to track. Alternatively, could add the Type as the second column instead of the last.

screen shot 2018-04-28 at 9 51 12 am

**Comment by meinstein** [comment]
font styling missing from blank prop state: 

screen shot 2018-04-28 at 10 19 43 am

**Comment by meinstein** [comment]
ux thought: what about filters for each column in the prop table? defaulting to alphabetical order based on prop name makes perfect, but would be nice, for example, to quickly re-order based on what props are required

**Comment by AlecAivazis** [comment]
I had that exact thought - the required ones are way more important to call out

**Comment by meinstein** [comment]
oh, and what about linking to the component's src code on github?

**Review by meinstein — APPROVED** [review]

**Pull request merged**